internal/cli/atlas/datalakepipelines/availablesnapshots/list.go

```diff
@@ -33,7 +33,7 @@ import (
 )
 
 var listTemplate = `ID	DESCRIPTION	STATUS{{range .Results}}
-{{if .DiskBackupReplicaSet}}{{.DiskBackupReplicaSet.Id}}	{{.DiskBackupReplicaSet.Description}}	{{.DiskBackupReplicaSet.Status}}{{else}}{{.DiskBackupShardedClusterSnapshot.Id}}	{{.DiskBackupShardedClusterSnapshot.Description}}	{{.DiskBackupShardedClusterSnapshot.Status}}{{end}}{{end}}`
```

Comment:
Good example on challenges we were getting with templates and polymorphic objects.